src/Symfony/Component/HttpKernel/Controller/TraceableControllerResolver.php

```diff
@@ -49,4 +56,26 @@ public function getController(Request $request)
 
         return $ret;
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @deprecated This method is deprecated as of 3.1 and will be removed in 4.0.
+     */
+    public function getArguments(Request $request, $controller)
+    {
+        @trigger_error(sprintf('This %s method is deprecated as of 3.1 and will be removed in 4.0. Please use the %s instead.', __METHOD__, TraceableArgumentResolver::class), E_USER_DEPRECATED);
```

Comment:
Woops, a small typo, `this` should be `the`, will fix this after travis is done as there was an error which I failed to see